tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
+from database_config import get_database_config
+
+
+class TestDatabaseConfigOptions:
+    def clean_env(self):
+        """
+        Setting different environment variables for test in the same process
+        can lead to flaky tests.
+        """
+        if "DATABASE_URL" in os.environ.keys():
+            del os.environ["DATABASE_URL"]
+
+        for key in filter(
+            lambda x: x.startswith("MXRUNTIME_Database"),
+            list(os.environ.keys()),
+        ):
+            del os.environ[key]
+
+    def test_no_setup(self):
+        self.clean_env()
+        try:
+            config = get_database_config()
+            assert config is None
+        except Exception as e:
+            assert "Can't find database configuration" in str(e)
```

Comment:
Asserting on exception messaging is not a very good idea in my opinion. Although @joshp123 recommended using a `RuntimeError`, it might be even better to create a custom exception class and use that.

Reply:
changed to context manager